### ISSUE TITLE
Inconsistent commit message on signoffs page

### ISSUE THREAD

**Issue by lahwaacz** [archlinux/archweb#533]
Right now the https://archlinux.org/packages/signoffs/ page shows this line:
```
python-pytest-xdist 3.6.1-1	any	Extra	arojas	1	Nov. 29, 2024	No	
Signoff
upgpkg: 3.6.1-2: Python 3.13 rebuild
```

Note that python-pytest-xdist 3.6.1-1 is in extra-testing, but the last column shows the commit message for version in 3.6.1-2 which is in extra-staging.

**Comment by jelly** [comment]
I don't think we can now unless do more complicated logic in https://github.com/archlinux/archweb/blob/c15b22203a3f2a10a371d5c1904da2185bb3d872/packages/management/commands/populate_signoffs.py#L55 now we just check out the last commit message, so it would need to match the tag somehow.

**Comment by lahwaacz** [comment]
Well the gitlab query provides all the tags, e.g. for python: https://gitlab.archlinux.org/api/v4/projects/archlinux%2Fpackaging%2Fpackages%2Fpython/repository/tags

I think something like this might work, provided we pass the right `version` of the package to show:

```python
    ...
    tags = r.json()

    # filter out unrelated tags
    tags = [tag for tag in tags if tag["name"] == version]

    if len(tags) == 0:
        logger.error("No tags found for pkgbase %s (%s) version %s", pkgbase, repo, version)
        return None

    tag = tags[0]
    ...
```

I think the version is in the `PackageSignoffGroup`, is it in the same format as the tag on GitLab?

https://github.com/archlinux/archweb/blob/c15b22203a3f2a10a371d5c1904da2185bb3d872/packages/utils.py#L271-L273
